app/assets/javascripts/uploadingDocument.js

```diff
@@ -19,7 +19,7 @@
                         setTimeout(refresh, parseInt(dataTag.dataset.interval));
                         break;
                     case "ERROR":
-                        goToErrorPage();
+                        window.location.href = "/problem-with-selected-file";
```

Comment:
All the other upscan routes are prefixed with file-upload - it would make sense if this one followed suit. Also I'd put our context route in the url used here just in case that url conflicts with something on another service for whatever reason.